Source/SwiftLintBuiltInRules/Rules/Lint/RedundantSendableRule.swift

```diff
@@ -36,8 +37,6 @@ struct RedundantSendableRule: Rule {
                     actor A: B, C // comment
                     {}
                     """),
-            Example("@MainActor protocol P: A, Sendable {}"):
-                Example("@MainActor protocol P: A {}"),
```

Comment:
Please keep this test case changing `protocol` to `struct` or `class`. It's important to ensure that the trailing comma is removed as well.